tests/test_extraction.py

```diff
@@ -1,31 +1,208 @@
-import unittest
-from textextraction import doc_process_toolkit as dpt
+import os
+from unittest import TestCase, skipUnless, main
+from textextraction.textextraction import (TextExtraction, PDFTextExtraction,
+                                           textextractor)
 
+ALL_INSTALLED = os.getenv("ALL_INSTALLED")
```

Comment:
I added this in so the tests don't break Travis, but maybe I'll just remove it from this repo.